editions/tw5.com/tiddlers/commands/Commands.tid

```diff
@@ -1,9 +1,15 @@
 created: 20150117174359000
-modified: 20180626122309578
+modified: 20241017175916756
 tags: Concepts Reference
 title: Commands
 type: text/vnd.tiddlywiki
 
 A <<.def command>> is one of the following words, written with a `--` prefix and used as a command-line option under Node.js, indicating which action is desired. See [[Using TiddlyWiki on Node.js]] for details of how to use them.
 
-<<list-links "[tag[Commands]]">>
+''List''
+
+<<list-links "[tag[Commands]] -[tag[$:/deprecated]]" class:"multi-columns">>
```

Comment:
I hadn't looked closely at the multi-columns class before. I think it needs some tweaks to keep things readable:

<img width="445" alt="image" src="https://github.com/user-attachments/assets/82d761ca-c367-40bb-bb7a-ca18d1616ac4">

Reply:
How did you create this screenshot? Below screen width 960px it should be 2 columns. So the screenshot seems to be just above that limit, with sidebar visible but "cut out" from the screenshot. 

IMO most mobile phone browsers will see 2 columns which is perfectly fine. At least on my phone.

We can change `multi-columns` to 3 columns, but that would look ugly with 1080p for most monitors today. There are several lists already in the core docs and **we did not get any complains about it** so far. So it seems to be "just right" for 2024.

The Commands tiddler is fine with 4 columns. Even just before the breakpoint.

Screenshot on 1080p monitor with 100% browser zoom and sidebar active. 

![image](https://github.com/user-attachments/assets/b707ec66-87a4-4b4d-82ee-90ec7be77cc2)